packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -32,6 +33,16 @@ export class AndroidSdkTools {
   private jdkHelper: JdkHelper;
   private pathJoin: (...paths: string[]) => string;
 
+  static async create(process: NodeJS.Process, config: Config, jdkHelper: JdkHelper,
+      log: Log = new ConsoleLog('AndroidSdkTools')): Promise<AndroidSdkTools> {
+    (await AndroidSdkTools.validatePath(config.androidSdkPath)).unwrap();
```

Comment:
I'm not quite clear on the intent of this line since we seem to be throwing away the result.

Are we just checking that the call to `unwrap` doesn't throw?

Reply:
`unwrap()` will cause the underlying error to be thrown, if the result is an `Error`. Otherwise, we ignore the response.

Reply:
Yes, we replaced a try and catch with that line..
validate path will return `Result<boolean>` if the path is valid, and  `Result<Error>` if it isn't